src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,58 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+
+        Manufacturer manufacturerTesla = new Manufacturer(null, "Tesla", "USA");
+        manufacturerService.create(manufacturerTesla);
+
+        Driver driverArtem = new Driver(null, "Artem Shevchenko", "12345");
+        driverService.create(driverArtem);
+
+        Driver driverAmina = new Driver(null, "Amina Franko", "56789");
+        driverService.create(driverAmina);
+
+        CarService carService =
+                (CarService) injector.getInstance(CarService.class);
+
+        Car carTeslaOne = new Car(null, "Tesla", manufacturerTesla, null);
+        carService.create(carTeslaOne);
+
+        Car carTeslaTwo = new Car(null, "Tesla", manufacturerTesla, null);
+        carService.create(carTeslaTwo);
```

Comment:
Consider creating a constructor without the 'id' parameter. In most cases, the 'id' will be generated automatically during persistence in the database, and it's uncommon to set it manually.